### DISCUSSION TITLE
skip_previous button missing when viewed via mobile

### DISCUSSION THREAD

**Discussion by bcambl** [BattlesnakeOfficial/feedback#152] [Bug Reports]
There seems to be a responsive design bug where the `skip_previous` button is missing when viewed on mobile (portrait).  
The button appears fine when in landscape mode.

```
<span class="material-icons icon-image-preview">skip_previous</span>
```
![Screenshot_20211217-113916](https://user-images.githubusercontent.com/5565939/146586865-d13f6ce9-f11f-49df-9919-a6dc5c1a3be9.png)

**Comment by bcambl** [comment]
![IMG_4302](https://user-images.githubusercontent.com/5565939/154090894-9afca30d-aeec-46c4-8158-5e61848fa119.jpg)

This issue appears to be fixed, Thank you! :tada:

**Reply by bvanvugt** [reply]
Yes! I managed to roll this out with the new scrubber feature :)
